Userland/Services/WebDriver/Session.cpp

```diff
@@ -158,4 +158,29 @@ ErrorOr<JsonValue, HttpError> Session::get_title()
     return JsonValue(m_browser_connection->get_title());
 }
 
+// POST /session/{session id}/refresh https://w3c.github.io/webdriver/#dfn-refresh
+ErrorOr<void> Session::refresh()
+{
+    // 1. If the current top-level browsing context is no longer open, return error with error code no such window.
+    auto current_window = get_window_object();
+    if (!current_window.has_value())
+        return HttpError { 400, "no such window", "Window not found" };
```

Comment:
404

Reply:
https://w3c.github.io/webdriver/#dfn-no-such-window

Reply:
than its wrong for the others, too. will fix.